Geometry_Engine/Query/IsContaining.cs

```diff
@@ -127,21 +167,37 @@ public static bool IsContaining(this Circle curve, List<Point> points, bool acce
 
         /***************************************************/
 
-        public static bool IsContaining(this Line curve1, List<Point> points, bool acceptOnEdge = true, double tolerance = Tolerance.Distance)
+        [PreviousInputNames("curve","curve1")]
```

Comment:
helps the UI ensure that the parameter is the same as it used to be.

Given this only has one input oif this particular type, it might be able to fallback to find it anyway, but rather not take that risk :)